### ISSUE TITLE
where should we make changes now?

### ISSUE THREAD

**Issue by markus2330** [sirblackheart/elektrasurvey#46]
1) questions.md
2) directly in limesurvey (which survey?)
3) or both?
4) import/export

**Comment by sirblackheart** [comment]
the work of keeping them in sync is rather tedious and error prone and needs constant documentation if both has happened.

We should probably decide for one of both and only sync them at the end of a cycle. I vote for using lime survey directly because we also get visual feedback on our changes.

**Comment by sirblackheart** [comment]
@Namoshek which survey should we use for the changes?

**Comment by Namoshek** [comment]
The one without anything in brackets at the end of the title... The one without (pilot). We will use the plain title for the final survey, that's why.

**Comment by sirblackheart** [comment]
I think you can still change the name afterwards, but it is a reason. So lets use the one without anything as our final target and the one renamed to have (Experiments) at the end to try out stuff.

**Comment by markus2330** [comment]
It might be useful to have one more pilot (maybe {pilot}), e.g. for the smileys. We could change current issues till tuesday/wednesday in this {pilot}.
Benefits would be:

- the changes from pilot to the real survey will be only minimal.
- On thursday we will look at it.
- we have some data with latest changes

What do you think?

**Comment by sirblackheart** [comment]
yes that is probably a good idea

**Comment by markus2330** [comment]
One problem is that it splits our statistics.

About the problem where to do the changes:

(5) what about modifying limesurvey (based on issues) and exporting some text file that we commit? (to refer to commits, review changes)

**Comment by markus2330** [comment]
Btw. I think one change was already done in questions.txt, please add it to limesurvey, before removing the file.

**Comment by sirblackheart** [comment]
> (5) what about modifying limesurvey (based on issues) and exporting some text file that we commit? (to refer to commits, review changes)

While it would help to keep track of changes here it also would create a work overhead especially for small changes. Isn't a comment that i did the changes in the issue enough?

**Comment by markus2330** [comment]
maybe something which is also displayed in github properly. What about doing a backup with all formats and then let us decide?

**Comment by sirblackheart** [comment]
ab7022ad44d9d633e508ce271fc8e6a8c95c200b

**Comment by markus2330** [comment]
xml is ok, questions_tab.txt is empty?

**Comment by markus2330** [comment]
does import work with both? xml looks a bit better.

Is there maybe some functionality for automatic export to github or something like that?

**Comment by sirblackheart** [comment]
> does import work with both? xml looks a bit better.

seems xml is not importable

> Select survey structure file (*.lss, *.csv, *.txt) or survey archive (*.lsa):

**Comment by markus2330** [comment]
ok, then xml is useless, can you export these formats too please?

**Comment by sirblackheart** [comment]
all possible already there
lls is ugly xml with a lot of cdata
i asume that txt and csv both refer to the tab seperated value format
survey archive is only possible at the end of a survey

**Comment by markus2330** [comment]
I am afraid thats a dead end.

So let us directly edit the Limesurvey with review of someone else if issue is now solved.

I updated the labels for the workflow:

1. discuss how to fix it
2. someone fixes it in limesurvey (and adds label)
3. someone reviews in limesurvey + says it is reviewed
4. I'll close the issues

is this workflow ok?

**Comment by markus2330** [comment]
Its not good that we have two variants now, 625192 and 624268, are both already modified?
Please delete one, there should be only one active and one for us to edit.

**Comment by sirblackheart** [comment]
I renamed 624268 back to experiments. It is there so we can try out stuff like changing the experience question. 625192 should be used for more final decisions.

**Comment by markus2330** [comment]
perfect, we fork the final survey then from 625192. Lets try to have 625192 up and running till tuesday.

I created a "Last Pilot" label for it. Hopefully we can fix every issue till then.

**Comment by Namoshek** [comment]
No, we will use 625192 as final survey and will fork a backup, because the scripts will not work in the backup without fixing them (due to different HTML IDs for the questions, etc.). Using a fork for the live-survey will result in less testing of it beforehand, so it is better to go with the one we already have (and it has the correct title by the way).

**Comment by markus2330** [comment]
So 625192 is the final one but which one should be updated now? Can you elaborate please? (Which one to use till tuesday, after tuesday and finally)

**Comment by Namoshek** [comment]
For final updates (decision that we change something is made), we use 625192 (the only survey without any additional note/scope in the title). For experiments (like testing the script for "to last page", see other issue), we use the one with `{experiments}`. The latter can also be used to try out things and see how they look and feel. Is this ok for you?

**Comment by markus2330** [comment]
I am okay with every number for every survey but it would be good to have some filled out surveys with the final corrections. So can we start the "final" survey on tuesday and somehow reset/change it later?

**Comment by markus2330** [comment]
I have updated the timeline in 2bcfb39b606925cfd0fa99f148ad17eaff73db95

Is this feasible for everyone? Which statistics does each of you want to make?

Would be good if at least one person plays around with R.
Does someone have SPSS available?

**Comment by Namoshek** [comment]
I saw the email and I did quite some search in the web, and although I know what is meant with the term `automatic rule learning`, I can't find any (general and helpful) information about the topic. Can you elaborate on what exactly you want me to do or give some hints, how to do it?

**Comment by markus2330** [comment]
I opened a new issue for the question #71
Please add new tools, methods and questions there.
